modules/ROOT/partials/configuration/noneditable_regexp.adoc

```diff
@@ -1,8 +1,9 @@
+[[noneditable_regexp]]
 === `noneditable_regexp`
 
 This option is used to specify a regular expression or array of regular expressions that {productname} will use to determine which areas of content are non-editable when using the `noneditable` plugin. The regular expressions needs to be global so that all instances within the document are matched. The text content of the matches will be wrapped in spans, hiding the structure and styling while editing.
 
-NOTE: If elements are matched by the regular expression, the elements will be replaced with spans. Use <<noneditable_noneditable_class,`noneditable_noneditable_class`>> for elements.
+NOTE: If elements are matched by the regular expression, the elements will be replaced with spans. Use xref:noneditable_noneditable_class[`noneditable_noneditable_class`] for elements.
```

Comment:
This xref target doesn't exist in this file so it should probably be something like this instead:
```suggestion
NOTE: If elements are matched by the regular expression, the elements will be replaced with spans. Use xref:plugins/opensource/noneditable.adoc#noneditable_noneditable_class[`noneditable_noneditable_class`] for elements.
```